oxenstored/connections.ml

```diff
@@ -122,7 +125,8 @@ let del_anonymous cons con spec_fds =
     cons.poll_status <-
       Array.make
         (Hashtbl.length cons.anonymous + List.length spec_fds)
-        (default_poll_status ()) ;
+        (default_poll_status ())
+    (* TODO: Work around an unnecessary allocation here *) ;
```

Comment:
Is this TODO something that needs to be resolved in this PR?

Reply:
Not particularly, just as a note to indicate a possible optimization. this could be solved if OCaml still allowed to create uninitialized arrays, for example, or through a more elaborate loop instead of the current two iterations. But I don't think this is a huge issue as the documentation indicates only one object is allocated and all the array elements just point back to it.